standards_lab/api/management/commands/migratedata.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from utils.project import get_project_config
+import os
+
+
+class Command(BaseCommand):
+    help = "Migrates files in storage to a new layout"
+
```

Comment:
Rename done.

Let me look at tests; I'm not sure how much effort to expand here given that it could be hard to write non-brittle tests and that we can actually delete this script in a months time anyway

Reply:
Ok, test done